packages/core/src/types/other.ts

```diff
@@ -3,11 +3,17 @@ export interface Statistic {
   metricValue: number;
 }
 
-export interface EvaluateResult {
+export interface EvaluateResultItem {
   pass?: boolean;
   [key: string]: any;
 }
 
+export interface EvaluateResult {
+  trainResult?: EvaluateResultItem;
```

Comment:
```suggestion
  train?: EvaluateResultItem;
```

The result might be duplicated with the interface name?